manager/integration/tests/test_basic.py

```diff
@@ -13,6 +15,47 @@
 from common import get_volume_endpoint, get_volume_engine
 from common import get_random_client
 from common import CONDITION_STATUS_FALSE, CONDITION_STATUS_TRUE
+from common import RETRY_COMMAND_COUNT
+
+
+def create_volume(client, vol_name, num_of_replicas=2,
+                  size=SIZE, base_image="", frontend="blockdev"):
+    client.create_volume(name=vol_name, size=size,
+                         numberOfReplicas=num_of_replicas,
+                         frontend=frontend, baseImage=base_image)
+    volume = common.wait_for_volume_detached(client, vol_name)
+    assert volume["name"] == vol_name
+    assert volume["size"] == size
+    assert volume["numberOfReplicas"] == num_of_replicas
+    assert volume["state"] == "detached"
+    assert volume["baseImage"] == base_image
+    assert volume["frontend"] == frontend
+    assert volume["created"] != ""
+    return volume
+
+
+def create_backup(client, volname):
+    volume = client.by_id_volume(volname)
+    volume.snapshotCreate()
+    data = write_volume_random_data(volume)
+    snap = volume.snapshotCreate()
+    volume.snapshotCreate()
+    volume.snapshotBackup(name=snap["name"])
+
+    bv, b = common.find_backup(client, volname, snap["name"])
+
+    new_b = bv.backupGet(name=b["name"])
+    assert new_b["name"] == b["name"]
+    assert new_b["url"] == b["url"]
+    assert new_b["snapshotName"] == b["snapshotName"]
+    assert new_b["snapshotCreated"] == b["snapshotCreated"]
+    assert new_b["created"] == b["created"]
+    assert new_b["volumeName"] == b["volumeName"]
+    assert new_b["volumeSize"] == b["volumeSize"]
+    assert new_b["volumeCreated"] == b["volumeCreated"]
+
+    return snap, data
```

Comment:
If the function called `create_backup`, it should return the backup as well. Probably backup volume too.